src/lib/content.js

```diff
@@ -177,7 +177,7 @@ function loadProfiles(profileSet, list, csrf, hidesHistory, hidesAccountId) {
   var AWSR_firstForm = null;
 
   document.getElementById('awsc-recent-roles-label').insertAdjacentHTML('beforeend', '<input id="AESR_RoleFilter" type="text" placeholder="Filter by profile name" style="border:1px solid #ccc;border-radius:3px;font-size:13px;margin-left:0.25em;max-width:20ex;padding:0.4ex">');
-
+  document.getElementById('awsc-username-menu-recent-roles').setAttribute("style", "height:200px;overflow:auto");
```

Comment:
@micaelgfilipe We need examine whether the 200px is appropriate and whether the scroll-bar width causes the style broken in Windows browsers. The height should be ideally changed according to the browser screen height by `calc(100vh - <absolute Y of account list>)`.

Reply:
I agree, and have improved. Regarding the possible Windows issue, I am not able to test it atm.